datasets/nrel-pds-building-stock.yaml

```diff
@@ -0,0 +1,39 @@
+Name: ComStock
+Description: |
+  The commercial building sector stock model, or ComStock, is a highly
+  granular, bottom-up model that uses multiple data sources, statistical
+  sampling methods, and advanced building energy simulations to estimate
+  the annual sub-hourly energy consumption of the commercial building stock
+  across the United States.
+Documentation: https://comstock.nrel.gov/
+Contact: ComStock@nrel.gov
+ManagedBy: '[National Renewable Energy Laboratory](https://www.nrel.gov/)'
+UpdateFrequency: Annually
+Collabs:
+  ASDI:
+    Tags:
+      - energy
+Tags:
+  - aws-pds
+  - energy
+  - sustainability
+License:  [ComStock License](https://github.com/NREL/ComStock/blob/main/LICENSE.txt)

```

Comment:
@MRossol I think this should pass if you get rid of the extra space and wrap in quotes.

Reply:
Thanks @jflasher! I couldn't figure out how to see the CI report, though I was on vacation and not trying very hard... Fix pushed.